main.py

```diff
@@ -128,7 +131,11 @@ def on_all_disconnect(self, widget):
     def on_fastboot_click(self, widget):
         subprocess.run(['adb', 'reboot', 'bootloader'], stdout=subprocess.PIPE).stdout.decode('utf-8')
         dialog = self.Finished(self)
-
+        
+    def on_fastbootd_click(self, widget):
+        subprocess.run(['fastboot', 'reboot', 'fastboot'], stdout=subprocess.PIPE).stdout.decode('utf-8')
```

Comment:
Would be better if you could add checks with previous functions to find if the device is already in fastboot or not.

Reply:
okay, i do understand your point, but how can one check for device is in fastboot mode or no. Any references. Thanks, for previous change, I'm amending it now. Thank You

Reply:
for now I guess that can be skipped, it's complicated to be honest since we don't use Google's adb and fastboot python modules
you can try this after merge, check if output of fastboot devices list anything, if it returns null then there's no device, if it returns more then it's present, need to check on tool sometime, will let you know

Reply:
Thank You Sir for generously helping. , If i get any idea on that. I'll ping you. Good Day.